src/main/java/info/smart_tools/smartactors/core/impl/SMObject.java

```
@@ -47,4 +48,14 @@ public void setValue(FieldName name, Object newValue) throws ChangeValueExceptio
         }
         map.put(name.toString(), newValue);
     }
+
+    @Override
+    public String toString() {
+        try {
+            return objectMapper.writeValueAsString(map);
+        } catch (JsonProcessingException e) {
+            // this method must not throw any exceptions
```

Great. But If we catch an JsonProcessingException we should throw own Exception in this case. It looks like we must throw something likes Error exception in this case. In other words we should not have any empty catch block of try-catch. There are no warranty that this exception will came again in a future.